### PR TITLE
Bugfix for dispatching events with correct uuid

### PR DESCRIPTION
**Pull request by Fsmash**
See https://github.com/dasundev/livewire-dropzone/issues/31

### CODE REVIEW

File: resources/views/livewire/dropzone.blade.php [hunk 1]
```diff
@@ -2,7 +2,8 @@
     x-cloak
     x-data="dropzone({
         _this: @this,
-        multiple: @js($multiple)
+        multiple: @js($multiple),
+        uuid: '{{ $uuid }}'
```
Comment (dasundev) [inline resources/views/livewire/dropzone.blade.php:6]: Instead of `uuid: '{{ $uuid }}'`, you can use `uuid: @js($uuid)`.

**Review by dasundev — CHANGES_REQUESTED** [review]
There are a few more issues:

1. I uploaded one file for each dropzone simultaneously, when deleting starting from the first dropzone, the file gets replaced with the file from the second dropzone.
2. Additionally, when each dropzone accepts multiple files, the uploading process does not work correctly.

File: resources/views/livewire/dropzone.blade.php (hunk 1 above)
Comment (Fsmash) [reply resources/views/livewire/dropzone.blade.php:6]: I ran into some weird issues like that when I didn't specify a key. By specifying ":key" essentially every issue I encountered went away. The thing to note is the key must be unique. I did this by just appending an extra character if you have multiple components being keyed in a loop like I do.  Let me know if this fixes things.
 
image

Comment (dasundev) [reply resources/views/livewire/dropzone.blade.php:6]: I tried with `:key`, it didn't solve the mentioned problems from my end.

Comment (Fsmash) [reply resources/views/livewire/dropzone.blade.php:6]: Odd. I am running into none of those issues. The dropzone component has been working pretty flawlessly after making those fixes. 

I can't show too much of the code I am working on but when I have the time, I will create a small example laravel project to show I am implementing this.

Comment (dasundev) [reply resources/views/livewire/dropzone.blade.php:6]: Everything looks good. The issue was on my end. I'll make the rest of the changes and merge the PR.

**Pull request merged**